python/grass/script/tests/test_script_task.py

```diff
@@ -1,11 +1,56 @@
+import os
+
+import pytest
+
+import grass.script as gs
 from grass.script.task import grassTask
 
 
-def test_mapcalc_simple_e_name():
+@pytest.fixture
+def xy_session_patched_env(tmp_path, monkeypatch):
+    """Active session in an XY location (scope: function), patching env vars directly.
+
+    This allows functions not accepting an env dictionnary argument to work in tests"""
```

Comment:
```suggestion
    This allows functions not accepting an env dictionary argument to work in tests"""
```

Reply:
Oops, totally right. I didn't realize it, as in French, "dictionnaire" has two Ns, so it didn't look wrong when reading it.